src/main/java/mate/academy/springboot/datajpa/dto/mapper/Mapper.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.springboot.datajpa.dto.mapper;
+
+public interface Mapper<T, Q, V> {
+    default V toResponseDto(T model) {
+        return null;
+    }
+
+    default T toModel(Q requestDto) {
```

Comment:
why default? it's not ok by default to return null, every implementation of this interface should implement both these methods

Reply:
Because, if i'll add new model, and need for it only one convertion (toModel or toResponseDto) - no need to implement another one. May be better to create distinct interface for each method...